spec/factories/case/overturned_icos.rb

```diff
@@ -1,11 +1,12 @@
 FactoryBot.define do
 
-  factory :overturned_ico_sar, class: Case::OverturnedICO::SAR do
-
+  factory :overturned_ico_sar,
+          aliases: [:ot_ico_sar_noff_unassigned],
+          class: Case::OverturnedICO::SAR do
     transient do
-      creation_time   { 4.business_days.ago }
-      identifier      { 'new overturned ico sar case' }
-      managing_team   { find_or_create :team_dacu }
+      creation_time   { Time.now }
+      # creation_time   { 4.business_days.ago }
```

Comment:
remove?

Reply:
Yup. Snuck in at the end there.

Reply:
Fixed. I think that that'll sort out the tests too. At least the filter-related ones.